scss/styles.scss

```diff
@@ -1,23 +1,73 @@
 .container {
   display: flex;
   flex-direction: column;
-  padding: 100px;
+  padding: 150px;
 }
 
 .title {
   margin-bottom: 30px;
 }
 
-.techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-bottom: 20px;
+.container-frontend {
+  align-self: center;
+  display: grid;
+  grid-template-columns: 260px 260px 260px;
+  grid-template-rows: 350px 350px;
+  grid-template-areas: 
```

Comment:
You can move all the properties  except this one to the container class to avoid repeating them in container backend